index.html

```diff
@@ -1512,113 +1499,146 @@ <h1>WoT Architecture</h1>
+            </p>
             <p>
-                A <a>Thing</a> in the Web of Things is the abstraction
-                of a physical or a virtual entity whose metadata and
-                interfaces are described by a WoT Thing Description. A
-                virtual entity is the composition of one or more Things.
-            </p>
-            <p>A simple example is a Thing being the digital
-                representation of a physical IoT devices, which also
-                directly provides the interface for interaction. A Thing
-                may also be hosted on a gateway or the cloud,
-                representing a physical entity with which clients cannot
-                communicate directly (e.g., a legacy device) or not all
-                (e.g., a room or table). Virtual entities like a room
-                are often represented by a composition of one or more
-                sensors and actuators, each of which may also be
-                available as an individual Thing.</p>
+                Drawn from this hypermedia principle,
+                the Web of Things defines <a>Interaction Affordances</a> as metadata of a Thing
+                that shows and describes the possible choices to <a>Consumers</a>, thereby suggesting how <a>Consumers</a> may interact with the <a>Thing</a>.
+                A general <a>Interaction Affordance</a> is navigation, which is activated by following a link, thereby enabling <a>Consumers</a> to browse the Web of Things.
+                <a href="#sec-interaction-model"></a> defines three more types of Interaction Affordances for W3C WoT: <a>Properties</a>, <a>Actions</a>, and <a>Events</a>.
+            </p>
             <p>
-                The interface of a <a>Thing</a> must follow the
-                interaction model of the WoT Architecture. However, <a>Things</a>
-                may only consist of metadata without offering an
-                interface for interaction.
+                Overall, this W3C WoT definition is aligned with HCI and interaction designers, who create physical Things,
+                as well as the REST and microservice community, who is working on Web services in general.
             </p>
         </section>
-        <section id="sec-arch-aspects">
-            <h2>Architectural Aspects of a Web Thing</h2>
+        <section id="sec-web-thing">
+            <h2>Web Thing</h2>
             <p>
-                The use cases and the requirements discussed result in
-                an abstract model of a Thing that has four architectural
-                aspects of interest: its behavior, its interactions, the
-                mapping of those interactions to concrete protocols
-                (what we call the Protocol Binding), and its security
+                A Web Thing has four architectural
+                aspects of interest: its behavior, its <a>Interaction Affordances</a>, and its security
```

Comment:
Please add Protocol Bindings to add the 4th aspect to this sentence

Reply:
This is not from me and I am waiting for @mmccool in #220